dch/package.json

```diff
@@ -14,6 +14,8 @@
   "author": "Michael Mitchell-Halter",
   "license": "WTFPL",
   "dependencies": {
-    "express": "~4.13.4"
+    "express": "~4.13.4",
+    "mongoose": "4.4.5", 
+    "mqtt": "1.7.3"
```

Comment:
This might cause a merge conflict with the automated test branch after its merged in. We can fix this later.